index.html

```diff
@@ -159,6 +159,15 @@ <h4 class="item-link__title">Open edX Learner's Guide</h4>
                     </div>
                   </a>
                 </li>
+                
+                <li class="item-link">
+                  <a class="item-link__action" href="http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/">
+                    <h4 class="item-link__title">Open edX Developer's Guide</h4>
+                    <div class="item-link__copy">
+                      <p>For developers contributing to the edX platform.</p>
```

Comment:
maybe we could make it more apparent that there's only one version:
Information for developers making contributions to the edX platform (latest version).
and then repeat on line 212 without the parenthetical.